src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <Authors>OpenTelemetry authors</Authors>

```

Comment:
This is already defined at `Common.prod.props`:
```suggestion
```

https://github.com/open-telemetry/opentelemetry-dotnet/blob/cb0630c8f9aeb0b937f05538420867d9d9e5e37d/build/Common.prod.props#L34